lib/src/http/managers/application_command_manager.dart

```diff
@@ -57,6 +59,8 @@ abstract class ApplicationCommandManager extends Manager<ApplicationCommand> {
       defaultMemberPermissions: maybeParse(raw['default_member_permissions'], (String raw) => Permissions(int.parse(raw))),
       hasDmPermission: raw['dm_permission'] as bool?,
       isNsfw: raw['nsfw'] as bool?,
+      integrationTypes: (raw['integration_types'] as List).cast<int>().map(ApplicationIntegrationType.parse).toList(),
+      contexts: (raw['contexts'] as List? ?? []).cast<int>().map(InteractionContextType.parse).toList(),
```

Comment:
```suggestion
      integrationTypes: parseMany(raw['integration_types'] as List, ApplicationIntegrationType.parse),
      contexts: maybeParseMany(raw['contexts'], InteractionContextType.parse),
```
The `contexts` field will also have to be made nullable.

Reply:
> [Interaction context(s)](https://canary.discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-interaction-context-types) where the command can be used, **only for globally-scoped commands**. By default, all interaction context types included for new commands.
(emphasis mine)

I expect this means the docs will be changed to make it nullable. I guess we can see for now. Should we open an issue for this on the discord/discord-api-docs repo?